modular_skyrat/modules/mutants/code/mutant_species.dm

```diff
@@ -0,0 +1,160 @@
+#define REGENERATION_DELAY 10 SECONDS  // After taking damage, how long it takes for automatic regeneration to begin
+
+/datum/species/mutant
+	name = "High-Functioning mutant"
+	id = "mutant"
+	say_mod = "moans"
+	meat = /obj/item/food/meat/slab/human/mutant/zombie
+	species_traits = list(NOBLOOD,NOZOMBIE,HAS_FLESH,HAS_BONE,NOEYESPRITES,LIPS,HAIR)
+	inherent_traits = list(TRAIT_ADVANCEDTOOLUSER,TRAIT_NOMETABOLISM,TRAIT_TOXIMMUNE,TRAIT_RESISTCOLD,TRAIT_RESISTHIGHPRESSURE,TRAIT_RESISTLOWPRESSURE,TRAIT_RADIMMUNE,TRAIT_LIMBATTACHMENT,TRAIT_NOBREATH,TRAIT_NODEATH,TRAIT_FAKEDEATH,TRAIT_NOCLONELOSS)
+	inherent_biotypes = MOB_UNDEAD|MOB_HUMANOID
+	mutanttongue = /obj/item/organ/tongue/zombie
+	var/static/list/spooks = list('sound/hallucinations/growl1.ogg','sound/hallucinations/growl2.ogg','sound/hallucinations/growl3.ogg','sound/hallucinations/veryfar_noise.ogg','sound/hallucinations/wail.ogg')
+	disliked_food = NONE
+	liked_food = GROSS | MEAT | RAW
+	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | ERT_SPAWN
+	bodytemp_normal = T0C // They have no natural body heat, the environment regulates body temp
+	bodytemp_heat_damage_limit = FIRE_MINIMUM_TEMPERATURE_TO_SPREAD // Take damage at fire temp
+	bodytemp_cold_damage_limit = MINIMUM_TEMPERATURE_TO_MOVE // take damage below minimum movement temp
+	limbs_icon = 'modular_skyrat/modules/mutants/icons/mutant_parts_greyscale.dmi'
+
+/datum/species/mutant/check_roundstart_eligible()
+	if(SSevents.holidays && SSevents.holidays[HALLOWEEN])
+		return TRUE
+	return ..()
+
+/mob/living/carbon/human/species/mutant
+	race = /datum/species/mutant
+
+/mob/living/carbon/human/species/mutant/infectious
+	race = /datum/species/mutant/infectious
+
+/datum/species/mutant/infectious
+	name = "Mutated Abomination"
+	mutanthands = /obj/item/mutant_hand
+	armor = 50
```

Comment:
hey so with tg's new change to armor this means that the base armor will make hollow points will do zero damage due to doubling the armor effect to 100

Reply:
Yee, I've taken that into account and rebalanced them, they come in two types, fast mutants and slow ones.